docs/csharp/whats-new/csharp-10.md

```diff
@@ -119,7 +119,7 @@ int x = 0;
 Prior to C# 10, there were many scenarios where definite assignment and null-state analysis produced warnings that were false positives. These generally involved comparisons to boolean constants, accessing a variable only in the `true` or `false` statements in an `if` statement, and null coalescing expressions. These examples generated warnings in previous versions of C#, but don't in C# 10:
 
 ```csharp
-string? representation;
+string representation;
```

Comment:
I think a better change here is to initialize this to a non-null value. That way, the undesired error is more clear. The new analysis correctly determines that `obj` is not null in the `true` branch:

```suggestion
string representation = "N/A";
```